ankh/main.go

```diff
@@ -190,6 +190,12 @@ func main() {
 			Desc:   "The kube config to use when invoking kubectl",
 			EnvVar: "KUBECONFIG",
 		})
+		contextOverride = app.String(cli.StringOpt{
+			Name:   "context",
+			Value:  "",
+			Desc:   "The context use. Overrides `current-context` in your Ankh config",
```

Comment:
I think this should be "The context to use."

Reply:
true, fixed.